src/Avalonia.Input/TextInput/InputMethodManager.cs

```diff
@@ -91,12 +92,15 @@ public void SetFocusedElement(IInputElement? element)
             _focusedElement = element;
 
             var inputMethod = (element?.VisualRoot as ITextInputMethodRoot)?.InputMethod;
-            if (_im != inputMethod)
-                _im?.SetActive(false);
-
+            
```

Comment:
It was needed here to forcefully deactivate IM

Reply:
fixed.